nmrfx-structure/src/test/python/runtests.py

```diff
@@ -1,15 +0,0 @@
-import unittest
-import java.lang.RuntimeException
-import sys
-
-#all test files have to be loaded into the current namespace to be able to
-# execute them with unittest.main(), so we have to do: "from x import *"
-#jython script
-from teststruct import *
-
-if __name__ == "__main__":
-    unittest.main()
```

Comment:
I am a bit confused by the naming of the files, in the processor test python directory there is a file runtests.py, but in this structure test python directory the runtests.py is deleted. Does runtests.py just run all the test scripts in the directory? and this is not needed here because there is only one python test script in structures?

Reply:
These were leftover from old way of running python tests.  I've now removed the runtests.py script (and a couple others that should have been removed)